src/components/BannerBlock.tsx

```diff
@@ -0,0 +1,54 @@
+import styled from "styled-components";
+
+interface BannerBlockProps {
+    content: {
+        text: string;
+        button: {
+            content: string;
+            link: string;
+        }
+    }
+}
+
+const BannerBlock = (props: BannerBlockProps) => (
+    <ApplicationDiv>
+      <FlexContainer>
+        <Text>{props.content.text}</Text>
+      </FlexContainer>
+      <FlexContainer>
+        <Button onClick={() => {}}>{props.content.button.content}</Button>
+      </FlexContainer>
+    </ApplicationDiv>
+);
+
+const ApplicationDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  gap: 10vw;
+  background-color: ${(props) => props.theme.colors.offWhite};
+`;
```

Comment:
i forgot to mention: can we add some padding here? at least 16px, up to 32px

Reply:
20px looks good so i set it to that